File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/PaymentAllocationDetails.cs
namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class PaymentAllocationDetails : Common.Entities.AuditDetails
    {
        public int AllocationId { get; set; }
        public string InvoiceType { get; set; }
        public decimal TotalAmount { get; set; }
        public int PayeeId { get; set; }
        public string PayeeName { get; set; }
        public string PayeeType { get; set; }
        public int AllocationStatusId { get; set; }
        public string AllocationStatus { get; set; }
        public int? MedicalInvoiceId { get; set; }
        public decimal AssessedAmount { get; set; }
        public decimal AssessedVat { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/medicare/src/app/medical-invoice-manager/models/payment-allocation-details.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';

export class PaymentAllocationDetails extends BaseClass {
    allocationId: number;
    invoiceType: string;
    totalAmount: number;
    payeeId: number;
    payeeName: string;
    payeeType: string;
    allocationStatusId: number;
    allocationStatus: string;
    medicalInvoiceId: number;
    assessedAmount: number;
    assessedVat: number;
}